.github/workflows/ci.yml

```diff
@@ -43,9 +45,50 @@ jobs:
         components: clippy, rustfmt
     - name: Cargo Test
       run: cargo test --all-features --workspace
-    - name: Cargo Bench
-      run: cargo bench
```

Comment:
This step is missing. We need to update `gh-worflow-tailcall` to support running benchmarks also.